src/js/services/forms.js

```diff
@@ -52,11 +52,13 @@ export default App.extend({
     const updated = dayjs().format();
     try {
       store.set(this.getStoreId(), { submission, updated });
+      this.trigger('submissionUpdated', updated);
```

Comment:
This is used by the `form_app.js` and `form-patient_app.js` to update the last updated time/date. Or it will cause that content to show for the first time if it's the first submission stored in local storage for that form.